simulation/traffic_simulator/src/data_type/data_types.cpp

```diff
@@ -16,6 +16,47 @@
 
 namespace traffic_simulator
 {
+namespace behavior
+{
+std::ostream & operator<<(std::ostream & stream, const Request & value)
+{
+  switch (value) {
+    case Request::NONE:
+      stream << "request : NONE" << std::endl;
+      break;
+    case Request::LANE_CHANGE:
+      stream << "request : LANE_CHANGE" << std::endl;
+      break;
+    case Request::FOLLOW_LANE:
+      stream << "request : FOLLOW_LANE" << std::endl;
+      break;
+    case Request::WALK_STRAIGHT:
+      stream << "request : WALK_STRAIGHT" << std::endl;
+      break;
+  }
+  return stream;
+}
+
+std::string getRequestString(const Request & value)
+{
+  switch (value) {
+    case Request::NONE:
+      return "none";
+      break;
+    case Request::LANE_CHANGE:
+      return "lane_change";
+      break;
+    case Request::FOLLOW_LANE:
+      return "follow_lane";
+      break;
+    case Request::WALK_STRAIGHT:
+      return "walk_straight";
+      break;
+  }
```

Comment:
The break after return is unnecessary, so delete it.

Reply:
**This less important change request was ignored because this pull request is a blocker for other important pull requests.**